src/pages/post-outwards/index.js

```diff
@@ -239,14 +348,31 @@ const Postoutwards = () => {
             setData={setData}
             rowId={['recordId']}
             pageSize={50}
-            pagination={!!formik.values.countryId}
+            pagination={true}
             paginationType='client'
             refetch={refetch}
             isLoading={false}
             maxAccess={access}
+            handleCheckboxChange={calcFc}
             showCheckboxColumn={true}
           />
         </Grow>
+        <Fixed>
+          <Grid container justifyContent='flex-end' spacing={2} sx={{ px: 2, mr: -11 }}>
```

Comment:
mr: -11? isn't there any better way to style this?

Reply:
i am not styling i am centering the total below the field